aptos-move/e2e-move-tests/src/tests/fee_payer.rs

```diff
@@ -58,6 +58,36 @@ fn test_existing_account_with_fee_payer() {
     assert!(bob_start > bob_after);
 }
 
+#[test]
+fn test_existing_account_with_fee_payer_aborts() {
+    let mut h = MoveHarness::new_with_features(vec![FeatureFlag::GAS_PAYER_ENABLED], vec![]);
```

Comment:
Do we need `SPONSORED_AUTOMATIC_ACCOUNT_V1_CREATION` flag also on?

Reply:
hmm... this is mostly for comments, because the default behavior is it is on...

Reply:
Will the test be OK when the default behavior changes? when the SPONSORED_AUTOMATIC_ACCOUNT_V1_CREATION is off?

Reply:
this is set in vm-genesis, if we change default flags, massive amounts of tests across the code base will start to fail not just in e2e-move-tests. We can do better here.

Reply:
eliminated the ambiguity...